WalletWasabi/WabiSabi/WabiSabiCoordinator.cs

```diff
@@ -28,7 +28,7 @@ public WabiSabiCoordinator(CoordinatorParameters parameters, IRPCClient rpc)
 
 			Arena = new(parameters.RoundProgressSteppingPeriod, rpc.Network, Config, rpc, Prison);
 
-			Postman = new(Config, Prison, Arena, Rpc);
+			Postman = new(Config, Prison, Arena);
```

Comment:
Is `Postman` some old name?

Reply:
No matter what the reason for that name was, it is always impossible for me to know what it is and I have to go and see what type it is in order to understand what it is for. 

Feel free to improve it.